test/ClassLoaderTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) Veaceslav Medvedev <slavcopost@gmail.com>
```

Comment:
Sure. The `phpStorm` defect, not permit to configure file templates for concrete project, only base for IDE :(